pywt/_extensions/_pywt.pyx

```diff
@@ -574,6 +607,255 @@ cdef public class Wavelet [type WaveletType, object WaveletObject]:
+        "Short wavelet family name"
+        def __get__(self):
+            return self.w.base.short_name.decode('latin-1')
+
+    property orthogonal:
+        "Is orthogonal"
+        def __get__(self):
+            return bool(self.w.base.orthogonal)
+        def __set__(self, int value):
+            self.w.base.orthogonal = (value != 0)
+
+    property biorthogonal:
+        "Is biorthogonal"
+        def __get__(self):
+            return bool(self.w.base.biorthogonal)
+        def __set__(self, int value):
+            self.w.base.biorthogonal = (value != 0)
+
+    property complex_cwt:
+        "CWT is complex"
+        def __get__(self):
+            return bool(self.w.complex_cwt)
+        def __set__(self, int value):
+            self.w.complex_cwt = (value != 0)
+
+    property lower_bound:
+        "Lower Bound"
+        def __get__(self):
+            if self.w.lower_bound != self.w.upper_bound:
+                return self.w.lower_bound
+        def __set__(self, float value):
+            self.w.lower_bound = value
+
+    property upper_bound:
+        "Upper Bound"
+        def __get__(self):
+            if self.w.upper_bound != self.w.lower_bound:
+                return self.w.upper_bound
+        def __set__(self, float value):
+            self.w.upper_bound = value
+
+    property center_frequency:
+        "Center frequency (shan, fbsp, cmor)"
+        def __get__(self):
+            if self.w.center_frequency > 0:
+                return self.w.center_frequency
+        def __set__(self, float value):
+            self.w.center_frequency = value
+
+    property bandwidth_frequency:
+        "Bandwidth frequency (shan, fbsp, cmor)"
+        def __get__(self):
+            if self.w.bandwidth_frequency > 0:
+                return self.w.bandwidth_frequency
+        def __set__(self, float value):
+            self.w.bandwidth_frequency = value
+
+    property fbsp_order:
+        "order parameter for fbsp"
+        def __get__(self):
+            if self.w.fbsp_order != 0:
+                return self.w.fbsp_order
+        def __set__(self, unsigned int value):
+            self.w.fbsp_order = value
+
+    property symmetry:
+        "Wavelet symmetry"
+        def __get__(self):
+            if self.w.base.symmetry == wavelet.ASYMMETRIC:
+                return "asymmetric"
+            elif self.w.base.symmetry == wavelet.NEAR_SYMMETRIC:
+                return "near symmetric"
+            elif self.w.base.symmetry == wavelet.SYMMETRIC:
+                return "symmetric"
+            elif self.w.base.symmetry == wavelet.ANTI_SYMMETRIC:
+                return "anti-symmetric"
+            else:
+                return "unknown"
+
+    def wavefun(self, int level=8, length = None):
+        """
+        wavefun(self, level=8, length = None)
+
+        Calculates approximations of scaling function (`phi`) and wavelet
+        function (`psi`) on xgrid (`x`) at a given level of refinement.
+
```

Comment:
remove reference to phi in this docstring  (looks like a copy/paste error from the Discrete case)